contract/metadata.go

```diff
@@ -1,6 +1,7 @@
 package contract
 
 type PutMetadataRequest struct {
+	PopulationType   string `schema:"population-type"`
 	DefaultDatasetID string `json:"default_dataset_id"`
```

Comment:
Please merge develop to get the latest changes, `population-type` is GetMetadataResponse is `population_type`